src/ForgeVTT.mjs

```diff
@@ -254,33 +254,13 @@ export class ForgeVTT {
         // the json data, since it can only be called once
         response.json = async () => result;
       }
+      console.log(`installPackage (${result.id})`, result);
```

Comment:
nitpick: console log without prefix

Either add prefix to this console log or remove it, as it seems like a debug one.